THIRD-PARTY-LICENSES.csv

```diff
@@ -1,74 +1,24 @@
 "module name","license","repository"
 "@angular-devkit/core@8.3.29","MIT","https://github.com/angular/angular-cli"
 "@angular-devkit/schematics@8.3.29","MIT","https://github.com/angular/angular-cli"
-"@angular/animations@17.3.12","MIT","https://github.com/angular/angular"
-"@angular/cdk@17.3.10","MIT","https://github.com/angular/components"
-"@angular/common@17.3.12","MIT","https://github.com/angular/angular"
-"@angular/compiler@17.3.12","MIT","https://github.com/angular/angular"
-"@angular/core@17.3.12","MIT","https://github.com/angular/angular"
-"@angular/forms@17.3.12","MIT","https://github.com/angular/angular"
-"@angular/material@17.3.10","MIT","https://github.com/angular/components"
-"@angular/platform-browser-dynamic@17.3.12","MIT","https://github.com/angular/angular"
-"@angular/platform-browser@17.3.12","MIT","https://github.com/angular/angular"
-"@angular/router@17.3.12","MIT","https://github.com/angular/angular"
+"@angular/animations@18.2.13","MIT","https://github.com/angular/angular"
+"@angular/cdk@18.2.14","MIT","https://github.com/angular/components"
+"@angular/common@18.2.13","MIT","https://github.com/angular/angular"
+"@angular/compiler@18.2.13","MIT","https://github.com/angular/angular"
+"@angular/core@18.2.13","MIT","https://github.com/angular/angular"
+"@angular/forms@18.2.13","MIT","https://github.com/angular/angular"
+"@angular/material@18.2.14","MIT","https://github.com/angular/components"
+"@angular/platform-browser-dynamic@18.2.13","MIT","https://github.com/angular/angular"
+"@angular/platform-browser@18.2.13","MIT","https://github.com/angular/angular"
+"@angular/router@18.2.13","MIT","https://github.com/angular/angular"
 "@braintree/sanitize-url@6.0.4","MIT","https://github.com/braintree/sanitize-url"
 "@datorama/akita@7.1.1","Apache-2.0","https://github.com/datorama/akita"
-"@fortawesome/angular-fontawesome@0.14.1","MIT","https://github.com/FortAwesome/angular-fontawesome"
+"@fortawesome/angular-fontawesome@0.15.0","MIT","https://github.com/FortAwesome/angular-fontawesome"
 "@fortawesome/fontawesome-common-types@0.3.0","MIT","https://github.com/FortAwesome/Font-Awesome"
-"@fortawesome/fontawesome-common-types@6.2.0","MIT","https://github.com/FortAwesome/Font-Awesome"
-"@fortawesome/fontawesome-svg-core@6.2.0","MIT","https://github.com/FortAwesome/Font-Awesome"
+"@fortawesome/fontawesome-common-types@6.7.2","MIT","https://github.com/FortAwesome/Font-Awesome"
+"@fortawesome/fontawesome-svg-core@6.7.2","MIT","https://github.com/FortAwesome/Font-Awesome"
 "@fortawesome/free-brands-svg-icons@6.0.0","(CC-BY-4.0 AND MIT)","https://github.com/FortAwesome/Font-Awesome"
 "@fortawesome/free-solid-svg-icons@6.0.0","(CC-BY-4.0 AND MIT)","https://github.com/FortAwesome/Font-Awesome"
-"@material/animation@15.0.0-canary.7f224ddd4.0","MIT","https://github.com/material-components/material-components-web"
```

Comment:
Going in order, did material repackage things or something? That's a a lot of deletions

Reply:
Oh, looks like they've moved to an internal fork 
https://blog.angular.dev/the-future-of-material-support-in-angular-7fa0662ecc4b